src/plugins/intel_gna/src/frontend/layer_quantizer.hpp

```diff
@@ -4,510 +4,81 @@
 
 #pragma once
 
-#include "ie_precision.hpp"
+#include "descriptions/gna_desc.hpp"
 #include "layers/gna_layer_info.hpp"
 #include "quantized_layer_params.hpp"
-#include "backend/gna_types.h"
 #include "quantization.hpp"
-#include "weights_converter.hpp"
-#include "gna_graph_tools.hpp"
 
+using namespace GNAPluginNS;
```

Comment:
I would remove using namespace from header.
If you need this for readability reason, because you use this in some method which body is placed in header, please use more specific using in context of this method.